package.json

```diff
@@ -70,6 +70,11 @@
 					"type": "boolean",
 					"default": true,
 					"description": "Forces indenting with two spaces when Dart files are opened. This is on by default because VS Code doesn't support per-language settings and most people use tabs/4 spaces for other languages."
+				},
+				"dart.showTodos": {
+					"type": "boolean",
+					"default": false,
```

Comment:
+1 to reanalyzing if that setting changes, it'll be nice to not have the UI
out of sync with the pref setting

On Aug 6, 2016 12:15 PM, "Danny Tuppeny" <notifications@github.com> wrote:

> In package.json
> <https://github.com/DanTup/Dart-Code/pull/38#discussion_r73790669>:
>
> > @@ -70,6 +70,11 @@
> >  					"type": "boolean",
> >  					"default": true,
> >  					"description": "Forces indenting with two spaces when Dart files are opened. This is on by default because VS Code doesn't support per-language settings and most people use tabs/4 spaces for other languages."
> > +				},
> > +				"dart.showTodos": {
> > +					"type": "boolean",
> > +					"default": false,
>
> I'm happy for the default to be on until we have a better option (and like
> you say, people more likely to look for an option if they don't like it).
> Not applying immediately sucks a little, but we could send an
> analysis.reanalyze from vscode.workspace.onDidChangeConfiguration?
> (though it looks like we don't get told which values changed; we'd have to
> always do it of stash a copy of the previous value)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DanTup/Dart-Code/pull/38/files/55b79d60afbf992d330e3f6636ef396ecf675393#r73790669>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABNg0YlEqVBazApmtZoNdi4gGW-zHcLYks5qdN1KgaJpZM4JeVbg>
> .
>